### ISSUE TITLE
This looks real interesting

### ISSUE THREAD

**Issue by tyeeman** [girobusan/mmds#1]
However, when I drag and drop the index.md file into Firefox I get this - 

```
Fatal error

Network down or host unreachable
```
If I click "edit" and type something new then save it the edit area goes blank. 
What am I doing wrong?

Update - OK I read the docs and started a server. Works now but looks like you missed enabling the sidebar in index.html. I copied from your docs and now I get the sidebar.

**Comment by girobusan** [comment]
Thank you! Yes, there was some issues with distribution files, take a look at new release — actually, it's the first one, which may be called "public release".

**Comment by JayGoldberg** [comment]
Agreed, interesting. I've been using [CMS.js](https://chrisdiana.github.io/cms.js/) which works just fine but would like it to be able to render posts from S3 or GCS or Google Drive. Have you considered forking that and continuing that project?

[Blogtini](https://github.com/traceypooh/blogtini) was another one.

**Comment by girobusan** [comment]
@JayGoldberg Thank you for pointing me to blogtini, it's brilliant idea. Just awesome. 

MMDS has very special set of restrictions. The main goal of it is an ability to quick  publish any set of markdown documents. Which can be used in other contexts (convert to pdf, drop to CMS, ...you name it) without editing, so:

- No front matter
- No restrictions on file names
- No server setup 

And that means, that it can not automatically build lists of pages. It's not that bad when your goal is to publish documentation or something alike, but lists are vital for more "web site like" approach. Or you'll need to add every page you've wrote to lists manually... nooo....

For example, I use MMDS for building "mini sites" for my students — I can edit and expand it in no time, I can use materials from there easily (as it is plain markdown), I can convert the site to pdf... and all of this literally in no time. That's a goal. For _websity_ web site I'd use some other tool.

Look also at my [IMP!](https://github.com/girobusan/imp) project — it can display page without JS and run embedded editor even without local server.

**Comment by JayGoldberg** [comment]
Gotcha. Yeah I'm just playing with CMS.js to do blogging, since I wanted to avoid a build environment etc, but still wanted to dynamically add pages and have them added to the site. I wanted tagging and search etc so the frontmatter is important to me.

**Comment by girobusan** [comment]
Looks like you can add pages dynamically to CMS.js. I'm thinking about some _method_ for maintaining lists of pages in unobtrusive way, but still have no ideas.
